gem_controllers/gem_controller.py

```diff
@@ -47,7 +47,8 @@ def make(
              speed controller.
             base_current_controller('PI'/'PID'/'P'/'ThreePoint'): Selection of the basic control algorithm for the
              current controller.
-            a(float):
+            a(float): Design parameter of the controller.
```

Comment:
That description is meaningless. Is there a correct technical term for this parameter? (@max-schenke ?)

Reply:
I assume that 'a' refers to the tuning parameter of the symmetrical optimum design method. If that is the case, I would be satisfied with naming it that way. Additionally, one could refer to some literature source where this method is explained,